DISCOVER/13_conference_culture.md

```diff
@@ -0,0 +1,234 @@
+
+### T-Shirts
+
+If you plan to offer t-shirts, consider offering more than one cut/fit of shirt.
+- Provide shirt sizing information with the registration form where you ask for shirt size. (Provide a wide range of sizes, including plus-size.)
+- Describe the cut in gender-neutral terms (e.g. &quot;fitted&quot; vs. &quot;straight-cut&quot;) with photo examples, rather than using gender-loaded terms (&quot;men&#39;s,&quot; &quot;women&#39;s,&quot; &quot;unisex&quot;).
+
+**Further reading:**
+
+- [Geek Feminism T-Shirts](http://geekfeminism.wikia.com/wiki/T-shirts)
+- [Unisex Swag Shirts Are Harming Your Community](http://cczona.com/unisex-swag-shirts-are-harming-your-community/)
+
+### Demographic Info
+
+- **Why do you need it?**
+  - We recommend you do not ask for information just &quot;out of curiosity&quot; but rather that there are **planned actions you will take depending on the results** of your demographic question(s).
+  - If you are asking just for the purpose of &quot;counting&quot; or ascertaining the diversity of your conference, we suggest doing this in the form of an anonymous post-survey instead, to protect participant privacy.
+- If you absolutely must include demographic info in your registration form, **make it optional, not required**. Forcing people to identify themselves by demographic category is not an inclusive practice and may prompt [stereotype threat](https://en.wikipedia.org/wiki/Stereotype_threat).
+- ✅ If you do collect demographic info, be thoughtful about the structure of the questions so that they are inclusive and don&#39;t contribute to alienating or &quot;othering&quot; marginalized people.
+  - Not having an answer choice that matches your identity can be a very alienating experience.
+- ✅ Take anonymity and data security seriously. Have a plan for limiting access to this information and keeping it secure.
+- Suggested format for demographic question:
+  - &quot;Do you self-identify as an underrepresented minority in either the [conference] community or in your professional field?&quot;
+    - &quot;Along which dimension(s) you self-identify as underrepresented? Check all that apply:&quot;
+         - Gender identity
+         - Ethnicity, nationality, skin color, race
+         - Sexual orientation
+         - Socioeconomic status
+         - Religion
+         - Age
+         - Ability
+         - Do not wish to provide
+         - Other (please specify)
+         
+**Further reading:**
+
+- [Guiding principles for handling gender on forms](http://43epnd.axshare.com/gender.html)
+- [A Word on Monitoring](https://www.cheryl-morgan.com/?p=24245)
+- [Respectful Collection of Demographic Data](https://medium.com/@anna.sarai.rosenberg/respectful-collection-of-demographic-data-56de9fcb80e2)
+- [Open Demographics](https://github.com/drnikki/open-demographics/tree/master/docs/_sources/questions)
+
+
+(pacman_rule)=
+## Pacman Rule
```

Comment:
none of the content in this section has anything to do with the Pacman rule. While a number of these things are reasonable, the header should be changed